dockers/squid/squid.conf

```diff
@@ -0,0 +1,8 @@
+acl SPICE_HOSTS dst demo.isardvdi.com
```

Comment:
Should the `dst` be hardcoded rather than loaded from the environment?

Reply:
it is parsed by script and modified